clients/smtpNotifier.go

```diff
@@ -39,15 +39,14 @@ func (c *SmtpNotifier) Send(to []string, subject string, msg string) (int, strin
 	}
 	mime := "MIME-version: 1.0;\nContent-Type: text/html; charset=\"UTF-8\";\n\n"
 	body := []byte("To: " + to[0] + "\r\n" +
-
 		"Subject: " + subject + "\r\n" +
-
 		mime + "\r\n" +
 		msg + "\r\n")
 	err := smtp.SendMail(c.Config.Server+":"+c.Config.Port, auth, c.Config.From, to, body)
 	if err != nil {
 		log.Println(err.Error())
 		return 400, err.Error()
 	}
+	log.Printf("SMTP email sent: %s\n", subject)
```

Comment:
Is it a good idea to log this kind of information? It may flood our logs...

Reply:
I don't know the amount of e-mails sent in production.
But the e-mails are sent only on specific event (account creation...), and not regularly to everybody.
There is a log message too in the current version.